### PR TITLE
Fix/cqlsh tests

### PR DESCRIPTION
**Pull request by nemines**
This PR should get the CQL tests to fail the build. Finally.

Includes a free gradle upgrade done in debugging phase.

### CODE REVIEW

**Comment by nemines** [comment]
This is now passing, but the build slowed down a lot due to force max-workers numbers and test heap size.
Experiments for better tuning / improvements here: https://github.com/palantir/atlasdb/pull/237

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServices.java [hunk 1]
```diff
@@ -186,6 +190,20 @@ public Void call() throws Exception {
         return peers;
     }
 
+    static class Local {
+        String data_center;
+        String rack;
+    }
```
Comment (clockfort) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServices.java]: nit spacing

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java [hunk 2]
```diff
@@ -1103,7 +1104,8 @@ public void putUnlessExists(String tableName, Map<Cell, byte[]> values)
     }
 
     String getFullTableName(String tableName) {
-        return configManager.getConfig().keyspace() + ".\"" + tableName + "\"";
+        String internalTableName = internalTableName(tableName);
+        return configManager.getConfig().keyspace() + ".\"" + internalTableName + "\"";
```
Comment (clockfort) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java]: nit could be one line

**Comment by clockfort** [comment]
Nice! Glad this worked, sorry it took so much time to debug.

+1'd, feel free to merge if you don't care about my nitpicks.

**Pull request merged**